src/Components/Message/Banner.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+	<div class="c-application c-banner" :class="[computedType]">
+		<div class="c-banner--img">
```

Comment:
```suggestion
		<div class="c-banner--image">
```
줄임말은 최대한 지양했으면 좋겠는데 어떻게 생각하시나요?
@hananumber1 @donghoon-song

Reply:
@yooseongsil full로 다쓰는게 좋을거 같긴하네요!

Reply:
동의합니다